dev-tools/travis/travis-script.sh

```diff
@@ -43,7 +43,7 @@ elif [ "$2" == "External" ]
 then
   if [ "$TRAVIS_JDK_VERSION" == "openjdk11" ]
   then 
-    TEST_MODULES='!storm-client,!storm-server,!storm-core,!storm-webapp,!storm-shaded-deps,!external/storm-cassandra,!external/storm-hive,!external/storm-hdfs,!external/storm-hbase,!sql/storm-sql-external/storm-sql-hdfs,!external/storm-hdfs-blobstore'
+    TEST_MODULES='!storm-client,!storm-server,!storm-core,!storm-webapp,!storm-shaded-deps,!external/storm-cassandra,!external/storm-hive,!external/storm-hdfs,!external/storm-hbase,!sql/storm-sql-external/storm-sql-hdfs,!external/storm-hdfs-blobstore,!external/storm-hdfs-login'
```

Comment:
Not sure why we have different `TEST_MODULES` on openjdk11 in first place. I added it here because `storm-hdfs-login` is similar to `storm-hdfs-blobstore`

Reply:
It's because cassandra and all of Hadoop don't run on Java 11 yet. See https://issues.apache.org/jira/browse/STORM-2798

Reply:
Thank you for clarification